src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -247,6 +248,35 @@
               + "cachable actions that output symlinks will fail.")
   public boolean allowSymlinkUpload;
 
+  /**
+   * Describes what kind of remote build outputs to download locally.
+   */
+  public enum RemoteOutputsStrategy {
```

Comment:
This enum is used outside of the `RemoteOptions` class, so I would suggest to move it out. Another argument for that would be to keep `RemoteOptions` class cleaner/simpler/smaller.

Reply:
done.